### PR TITLE
Updated mobile view background and also size of contact details

### PR DESCRIPTION
**Pull request by VasaviLagishetty**
mobile_view

### CODE REVIEW

File: css/grayscale.scss [hunk 1]
```diff
@@ -351,6 +351,19 @@ a{
     color: white;
 }
 
+@media(min-width:768px) {
```
Comment (SriSatyaLokesh) [inline css/grayscale.scss:354]: is this change related to contacts?

Comment (SriSatyaLokesh) [reply css/grayscale.scss:354]: may be we need this to be enlarged in web view.

Comment (VasaviLagishetty) [reply css/grayscale.scss:354]: Exactly what it is done there if the screen width is 786px or more it will enlarge.

**Review by SriSatyaLokesh — APPROVED** [review]
This is good. we can merge this to master.

File: css/grayscale.scss (hunk 1 above)
Comment (SriSatyaLokesh) [reply css/grayscale.scss:354]: got it. Thanks.
